experimental/Schemes/BlowupMorphism.jl

```diff
@@ -151,10 +161,69 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   inc_Z_trans = CoveredClosedEmbedding(Y, I_trans, 
                                        covering=simplified_covering(Y), # Has been set by the previous call
                                        check=false)
-  inc_cov = covering_morphism(inc_Z_trans)
+  inc_dom_cov = covering_morphism(inc_Z_trans)
+  inc_cod_cov = covering_morphism(inc)
 
   Z_trans = domain(inc_Z_trans)
   pr_res = restrict(projection(p), inc_Z_trans, inc)
+
+  if has_attribute(p, :isomorphism_on_open_subset)
+    OOX = OO(X)
+    OOY = OO(Y)
+    p_iso = isomorphism_on_open_subset(p)
+    U = domain(p_iso)::PrincipalOpenSubset
+    V = codomain(p_iso)::PrincipalOpenSubset
+    V_amb = ambient_scheme(V)
+    U_amb = ambient_scheme(U)
+
+    # We have the following diagram:
+    #             inc_dom
+    #     Z_trans    ↪     X   ⊃ U_amb ⊃ U
+    #
+    #  pr_res↓             ↓ p           ↓ p_iso
+    #        Z       ↪     Y   ⊃ V_amb ⊃ V
+    #             inc_cod
```

Comment:
The code  (lines 156 - 159) and the docstring suggest the following
```suggestion
    # We have the following diagram:
    #             inc_dom
    #     Z_trans    ↪     Y   ⊃ U_amb ⊃ U
    #
    #  pr_res↓             ↓ p           ↓ p_iso
    #        Z       ↪     X   ⊃ V_amb ⊃ V
    #             inc_cod
```